src/MiniProfiler.Shared/Timing.cs

```diff
@@ -77,28 +78,76 @@ public Timing(MiniProfiler profiler, Timing parent, string name, decimal? minSav
         /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
         /// </summary>
         [DataMember(Order = 5)]
-        public List<Timing> Children { get; set; }
+        public IImmutableList<Timing> Children
```

Comment:
so it looks like IImmutableList works with the protobuf library, however i don't see a good alternative to Dictionary, there is ImmutableDictionary but it has poor performance comparing to the regular dictionary, i think the best option is to keep List and Dictionary until protobuf is fixed